### DISCUSSION TITLE
I am using next js 14 how can I implement this package in it

### DISCUSSION THREAD

**Discussion by nikita-fxbytes** [i18next/next-i18next#2255] [Q&A]
npm install i18next react-i18next ni18n

**Comment by adrai** [comment]
ni18n is not next-i18next

**Reply by nikita-fxbytes** [reply]
I want to implement multilanguage functionality in my Next.js 14 application without having language codes in the URL. I want the selected language to persist even after page refreshes. How can I implement this functionality in Next.js 14? I am using app routing in Next.js.

**Reply by adrai** [reply]
https://github.com/i18next/next-app-dir-i18next-no-locale-path-example

**Comment by adrai** [comment]
https://locize.com/blog/next-app-dir-i18n/

**Comment by nikita-fxbytes** [comment]
Hello, I'm urgently seeking assistance to implement multilanguage functionality in my Next.js 14 application. I aim to have language selection without language codes in the URL, and I need the selected language to persist across page refreshes. Could someone please help me with this implementation? I'm using app routing in Next.js. Your prompt support would be greatly appreciated. Thank you!

**Reply by adrai** [reply]
https://github.com/i18next/next-app-dir-i18next-no-locale-path-example
